satpy/readers/aws_l1c.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2023, 2024 Pytroll Developers
```

Comment:
File naming: `mws_l1c.py`. Haven't yet gone in-depth to see should there be a common base for the L1b and L1c readers.

Reply:
Yes, there should probably, and a common base for the test functions.